test_conformance/vulkan/test_vulkan_interop_image.cpp

```diff
@@ -1299,38 +1313,29 @@ int test_image_common(cl_device_id device_, cl_context context_,
     cl_kernel kernel_unsigned[num_kernels] = { NULL, NULL, NULL, NULL };
```

Comment:
Since you are at it, can you also initialize program array above to NULL for the sake of consistency and completeness?